### ISSUE TITLE
Add ping_sitemap_indexers to admin page

### ISSUE THREAD

**Issue by nworden** [google/personfinder#556]
Currently you have to set it on the command line.

**Comment by kylejb** [comment]
@nworden / @gimite - I will be tackling this issue for my first contribution.

**Comment by nworden** [comment]
Cool, thanks! Let me know if you have any questions.

**Comment by kylejb** [comment]
Thanks! Would it be best to ask questions here or would you prefer another medium like Discord?

**Comment by nworden** [comment]
Here's good, we don't really use anything besides GitHub.

**Comment by kylejb** [comment]
No problem! 

I'm having some preliminary issues running the application locally. The error message pertains to a file titled "**lxml.py.**" It's missing from the directory. And the error message specifically reads "**ImportError: No module named common**".

Are you able to run it locally right now?

**Comment by nworden** [comment]
Yeah, I haven't had any problems running it locally recently. A couple questions to help debug this:
1) Do you see this error message right away, or when you try to visit a page?
2) Have you run `pip install -r requirements.txt -t app/vendors`? You might be missing lxml altogether.
3) Would you mind posting the whole stack trace?

**Comment by kylejb** [comment]
> Do you see this error message right away, or when you try to visit a page?

Only after I visit localhost:8000, I see the error message.

> Have you run `pip install -r requirements.txt -t app/vendors`? You might be missing lxml altogether.

I am definitely missing lxml altogether. But, I did run the aforementioned command. I was initially have some problems running it due to an apparently common conflict that `pip install` has with `brew install` python. I resolved it after finding a workaround online. The error and workaround for that issue is described [here](https://stackoverflow.com/questions/24257803/distutilsoptionerror-must-supply-either-home-or-prefix-exec-prefix-not-both). I had to add my own setup.cfg file on my system with the following lines:
`[install]
prefix=`
I, then, seemed to have successfully downloaded the required packages into the vendors folder, which is nested under the "apps" directory.

> Would you mind posting the whole stack trace?
See attached for my terminal output when attempting to running local.
[OpensourceProject_TerminalOutput.txt](https://github.com/google/personfinder/files/3266860/OpensourceProject_TerminalOutput.txt)

_FYI, tried this on several computers and Operating Systems. Let's assume, going forward, that I'll be working on latest MacOSX with Python3 and Python2 via Brew – in addition to the system default of Python2._

Thanks for the help. I naively thought that my challenge would solely pertain to the coding assignment rather than the prerequisites!

**Comment by nworden** [comment]
Oh, I forgot lxml is one of the non-vendored ones; did you also already run `sudo pip install pytest 'lxml>=3.5.0' cssselect oauth2client modernize mock pylint yapf`?
I haven't used brew so I can't offer you much help there, but fwiw some people (especially if they're also working on other Python projects) find it helpful to use [virtualenv](https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv).

**Comment by kylejb** [comment]
I did run that command, but not in virtualenv – virtualenv got it running locally. Thank you for the suggestion.

I'll check in with you on my progress later this weekend and will let you know if I have any additional questions.

**Comment by kylejb** [comment]
@nworden - 

I am finding the task to be beyond my current skill level. The biggest obstacle has been working through code written by others. As a beginner, I have only been exposed to Python in isolation. But, through this experience, I identified these skill gaps and will work on it.

As much as I relish the challenge and this awesome opportunity to progress while also helping your team with this incredible application, I foresee it will place an undue burden on you (and other reviewers). But, I would like to return when I am ready and another issue presents itself.

Thank you for the opportunity.

Oh, and if you run into an issue on a project that you think I might be well-suited for, feel free to drop me a line.

**Comment by nworden** [comment]
Ok, no worries Kyle. We'll be happy to have you back later if you want to return :)

**Comment by kylejb** [comment]
Thank you!
